composer.json

```diff
@@ -7,7 +7,7 @@
     "prefer-stable": true,
     "require": {
         "drupal/core": "^8.7",
-        "drupal/field_group": "^1.0",
+        "drupal/field_group": "^3.0.0-rc1",
```

Comment:
We actually need to drop this dependency as it's a leftover. We will move it to the ewcms.